### PR TITLE
Add installation instructions

### PR DESCRIPTION
**Pull request by rajdeepbharati**

### CODE REVIEW

**Comment by mojca** [comment]
Thank you very much for the documentation.

The following is not meant to be actual feedback to your work ... For some reason (probably had to do with the fact that the last time I actively used npm was a very long time ago) I had npm 5 installed and that one conveniently crashes with unhandled errors when trying to install packages. After upgrading to version 6 everything went smooth. I don't know if npm 5 is simply no longer supported by the set of packages used, or if it's just a transient error on my machine.

I'll try to finish the installation locally, but I assume this is ready to be merged.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,28 @@
 # buildbot-macports-custom-views
 
 Custom views plugin for MacPorts Buildbot.
+
+## Getting Started
+
+### Set up buildbot environment
+
+Set up a buildbot environment: http://docs.buildbot.net/latest/developer/quickstart.html
```
Comment (mojca) [inline README.md]: Just out of curiosity: does one really need to install buildbot in "full developer mode" as described under http://docs.buildbot.net/latest/developer/quickstart.html#create-a-buildbot-python-environment for the sake of using (or maybe slightly modifying) this plugin?

Comment (rajdeepbharati) [reply README.md]: > Just out of curiosity: does one really need to install buildbot in "full developer mode" as described under http://docs.buildbot.net/latest/developer/quickstart.html#create-a-buildbot-python-environment for the sake of using (or maybe slightly modifying) this plugin?

Apparently, installing buildbot in developer mode isn't needed.

Comment (rajdeepbharati) [reply README.md]: A better option is to use `python3 -m venv .venv`. I think this should work on mac and linux

Comment (mojca) [reply README.md]: As a replacement for https://github.com/buildbot/buildbot/blob/master/Makefile#L82 ?

Comment (rajdeepbharati) [reply README.md]: Yes, but it doesn't work with python2 (anyways buildbot doesn't use python2 now)

**Pull request merged**